proguard-slim.cfg

```diff
@@ -0,0 +1,16 @@
+-injars build/graphhopper.jar
+-libraryjars <java.home>/lib/rt.jar
+
+-dontoptimize
+-dontobfuscate
+-dontpreverify
+-printusage
+-dontnote
+
+-keep public class com.graphhopper.GHRequest, com.graphhopper.GHResponse, com.graphhopper.GraphHopper, com.graphhopper.ResponsePath, com.graphhopper.config.CHProfile, com.graphhopper.config.LMProfile, com.graphhopper.config.Profile, com.graphhopper.http.WebHelper, com.graphhopper.routing.lm.LMPreparationHandler, com.graphhopper.util.Helper, com.graphhopper.util.Instruction, com.graphhopper.util.InstructionList, com.graphhopper.util.PMap, com.graphhopper.util.Parameters, com.graphhopper.util.PointList, com.graphhopper.util.RoundaboutInstruction, com.graphhopper.util.Translation, com.graphhopper.util.TranslationMap, com.graphhopper.util.exceptions.PointNotFoundException, com.graphhopper.util.exceptions.PointOutOfBoundsException, com.graphhopper.util.shapes.GHPoint3D {
```

Comment:
A może by to sformatować tak by każda klasa była w nowej linijce? Będzie długi plik, ale łatwo można będzie ogarnąć co jest na tej whiteliście